docs/ARCHITECTURE.md

```diff
@@ -98,6 +153,14 @@ $text = AiClient::generateTextResult(
 
 _Note: Since this omits the model parameter, the SDK will automatically determine which models are suitable and use any of them, similar to [the first code example](#generate-text-using-any-suitable-model-from-any-provider-most-basic-example). Since it knows the input includes an image, it can internally infer that the model needs to not only support `AiCapability::TEXT_GENERATION`, but also `AiOption::INPUT_MODALITIES => ['text', 'image']`._
 
+##### Fluent API
+```php
+$text = AiClient::prompt('Generate alternative text for this image.')
+    ->withImageFile(new InlineFile('image/png', $base64blob))
```

Comment:
Based on https://github.com/WordPress/php-ai-client/pull/12#discussion_r2203525309, if we go with that, could you reverse the order here accordingly too?